rules-tests/DowngradePhp74/Rector/ClassMethod/DowngradeContravariantArgumentTypeRector/Fixture/phpdoc_method.php.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp74\Rector\ClassMethod\DowngradeContravariantArgumentTypeRector;
+
+/**
+ * @method InterfaceWithPhpdocMethod someMethod(string|null $param)
+ */
+interface InterfaceWithPhpdocMethod
+{
+}
```

Comment:
This should be moved to `/Source` directory as not part of the test, just used class.

Reply:
Done